### PR TITLE
adding spark notebook

### PR DESCRIPTION
**Pull request by ashwin-patil**
- Azure sentinel notebook leveraging synapse workspace connected to server less spark pool

### CODE REVIEW

**Comment by zhzhao8888** [comment]
One thing: Can we remove 'notebook' in the file name?  Kind of redundant.

**Comment by zhzhao8888** [comment]
"## Hunting on historical data at scale using Azure Synapse Spark notebook\r\n",
Can we remove 'notebook' in the file name?

"__Python Version:__ Python 3.6 - AzureML<br>\r\n",
Should be 3.8, correct?

"3. Configurate Azure ML and Azure Synapse Analytics\r\n",
Change Configurate to Configure

**Comment by ashwin-patil** [comment]
> One thing: Can we remove 'notebook' in the file name? Kind of redundant.

done.

**Comment by ashwin-patil** [comment]
> "## Hunting on historical data at scale using Azure Synapse Spark notebook\r\n", Can we remove 'notebook' in the file name?
> 
> "**Python Version:** Python 3.6 - AzureML\r\n", Should be 3.8, correct?
> 
> "3. Configurate Azure ML and Azure Synapse Analytics\r\n", Change Configurate to Configure

done

**Review by zhzhao8888 — APPROVED** [review]

File: Guided Hunting - Detect potential network beaconing using Apache Spark via Azure Synapse.ipynb [hunk 1]
```diff
@@ -0,0 +1,1451 @@
...
+      "cell_type": "markdown",
+      "source": [
+        "## Configure Azure ML and Azure Synapse Analytics\r\n",
+        "\r\n",
+        "Please use notebook [Configurate Azure ML and Azure Synapse Analytics](https://github.com/Azure/Azure-Sentinel-Notebooks/blob/master/Configurate%20Azure%20ML%20and%20Azure%20Synapse%20Analytics.ipynb) to configure environment. \r\n",
+        "\r\n",
+        "The notebook will configure existing Azure synapse workspace to create and connect to Spark pool. You can then create linked service and connect AML workspace to Azure Synapse workspaces.\r\n",
+        "<br>It will also configure data export rules to export data from Log analytics workspace CommonSecurityLog table to Azure Data lake storage Gen 2.\r\n",
+        "\r\n",
+        "> **Note**: Specify the input parameters in below step in order to connet AML workspace to synapse workspace using linked service."
```
Comment (chinguyen1) [inline Guided Hunting - Detect potential network beaconing using Apache Spark via Azure Synapse.ipynb:235]: Small typo on "connet" :)

File: Guided Hunting - Detect potential network beaconing using Apache Spark via Azure Synapse.ipynb [hunk 2]
```diff
@@ -0,0 +1,1451 @@
...
+        "__Data Source Required:__ Yes\r\n",
+        "\r\n",
+        "__Data Source:__ CommonSecurityLogs\r\n",
+        "\r\n",
+        "__Spark Version:__ 3.1 or above\r\n",
+        "    \r\n",
+        "### Description\r\n",
+        "In this sample guided scenario notebook, we will demonstrate how to set up continuous data pipeline to store data into azure data lake storage (ADLS) and \r\n",
+        "then hunt on that data at scale using distributed processing via Azure Synapse workspace connected to serverless Spark pool. \r\n",
+        "Once historical dataset is available in ADLS , we can start performing common hunt operations, create a baseline of normal behavior using PySpark API , Azure Sentinel watchlists and also apply data transformations \r\n",
```
Comment (chinguyen1) [inline Guided Hunting - Detect potential network beaconing using Apache Spark via Azure Synapse.ipynb:22]: Please remove "Azure Sentinel watchlists" so users won't expect it in this version

**Review by chinguyen1 — COMMENTED** [review]
@ashwin-patil I added a couple small comments. Thanks!

**Review by chinguyen1 — APPROVED** [review]

**Pull request merged**